dev/com.ibm.ws.jdbc_fat_oracle/publish/files/oracle-ssl/Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM kyleaure/oracle-18.4.0-xe-prebuilt:2.0
+FROM kyleaure/oracle-18.4.0-expanded:1.0.full
```

Comment:
No, the `FROM` part of this docker file describes the image to build on top of (or from).  I want to start with `1.0.full`, copy these files, run these scripts, etc. and then publish it as `1.0.full.ssl`

Reply:
The reason I had to use `1.0.full` and not `1.0.slim` was because the slim copy doesn't have java or orapki installed in order to create an oracle wallet / JKS keystores.